backend/src/graphql.ts

```diff
@@ -317,42 +340,39 @@ schema.mutationType({
         })
         return user
       },
-    }),
-      t.field('loginUser', {
-        type: 'User',
-        args: {
-          identifier: stringArg({ required: true }),
-          password: stringArg({ required: true }),
-        },
-        resolve: async (parent, args, ctx: any) => {
-          const user = await ctx.db.user.findOne({
-            where: {
-              email: args.identifier,
-            },
-            include: {
-              auth: true,
-            },
-          })
-          if (!user) {
-            throw new Error('User not found')
-          }
-
-          const isValid = await bcrypt.compare(
-            args.password,
-            user.auth.password,
-          )
-
-          if (!isValid) {
-            throw new Error('Invalid password')
-          }
-          const token = jwt.sign({ userId: user.id }, process.env.APP_SECRET!)
-          ctx.response.cookie('token', token, {
-            httpOnly: true,
-            maxAge: ONE_YEAR,
-          })
-          return user
-        },
-      })
+    })
+    t.field('loginUser', {
+      type: 'User',
+      args: {
+        identifier: stringArg({ required: true }),
+        password: stringArg({ required: true }),
+      },
+      resolve: async (parent, args, ctx: any) => {
+        const user = await ctx.db.user.findOne({
+          where: {
+            email: args.identifier,
+          },
+          include: {
+            auth: true,
+          },
+        })
+        if (!user) {
+          throw new Error('User not found')
+        }
+
+        const isValid = await bcrypt.compare(args.password, user.auth.password)
+
+        if (!isValid) {
+          throw new Error('Invalid password')
+        }
+        const token = jwt.sign({ userId: user.id }, process.env.APP_SECRET!)
+        ctx.response.cookie('token', token, {
+          httpOnly: true,
+          maxAge: ONE_YEAR,
```

Comment:
Here's at least one example of where merging this would be bring back in old stuff into master. `ONE_YEAR` has been removed entirely as mentioned in a comment up above. I am very worried about this merge, especially in this file and it's very hard to parse everything quickly with all the formatting changes and the way that they are... but this kind of thing (this example) is what I was worried about earlier when saying I am worried about this merge